src/pages/Header/TopMenu.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+import { IconButton, Menu } from '@mui/material';
+
+import { MenuContainer, StyledMenuIcon, StyledMenuItem } from './styled';
+import { ActiveModal } from '../../types/modal';
+
+type Props = {
+    menuList: ('Settings' | 'Info' | 'Stats' | null)[];
+    onMenuChange: (newActiveModal: ActiveModal) => void;
+};
+
+const TopMenu = ({ menuList, onMenuChange }: Props) => {
+    const [anchor, setAnchor] = useState<null | HTMLElement>(null);
+    const open = Boolean(anchor);
```

Comment:
boolean names should be question like eg. isMenuOpen in this case

Reply:
Fixed.